### PR TITLE
Fixes for eap-cluster-app

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.jboss.org/browse/JDG-1746

### CODE REVIEW

File: eap-cluster-app/build.sh [hunk 1]
```diff
@@ -19,10 +19,10 @@ case "$1" in
       #Unzipping the JDG EAP Modules to jdg-modules directory
       echo "Unzipping JDG Modules ..."
       unzip -q ${JDG_MODULES_ZIP_PATH} -d setupOutput/jdg-modules
-      export JDG_MODULES=`cd setupOutput/jdg-modules/*;pwd`
+      export JDG_MODULES=`cd setupOutput/jdg-modules;pwd`
```
Comment (karesti) [inline eap-cluster-app/build.sh]: are you sure this changes are necessary ? because this does not fail in testing environements

**Review by karesti — COMMENTED** [review]
I'm just wondering why build.sh does not fail in their env neither mine and fails in yours. I won't change the build.sh which is a touchy file

File: eap-cluster-app/build.sh (hunk 1 above)
Comment (ryanemerson) [reply eap-cluster-app/build.sh]: I can only go from what occurred on my machine, but without them the JDG modules were not copied to the server modules dir

Comment (ryanemerson) [reply eap-cluster-app/build.sh]: @karesti You were right these changes weren't necessary. The problem was that I used the modules zip built straight from source, which does not have the `modules` and `docs` subdirectories, hence why my change was needed on my machine. When using the released zip everything was fine, so I have removed the commit from this PR. Sorry for the confusion.

**Comment by karesti** [comment]
@ryanemerson integrated, thanks !!! 💃

**Pull request merged**